props/problem-set-timeline.md

```diff
@@ -99,18 +99,19 @@ export default App;
 ```css
 .App-header {
   background-color: #222;
-  padding-bottom: 0.5rem;
+  padding-top: 7rem;
+  padding-bottom: 1rem;
   color: white;
   width: 100%;
+  margin: 0rem;
 }
-
+​
```

Comment:
Zero-width spaces snuck into the code in here too.

A regex search in vscode for `[\u200B-\u200D\uFEFF]` should turn up any unusual characters.